cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/detector/TopicReplicationFactorAnomalyFinder.java

```diff
@@ -103,5 +187,40 @@ public void configure(Map<String, ?> configs) {
               TOPIC_REPLICATION_FACTOR_ANOMALY_CLASS_CONFIG, _topicReplicationTopicAnomalyClass));
       }
     }
+    try {
+      _topicReplicationFactorMargin = Short.parseShort((String) configs.get(TOPIC_REPLICATION_FACTOR_MARGIN_CONFIG));
```

Comment:
Should we have a sanity check here regarding the acceptable range of values in the `MARGIN`?